bin/lib/zopen-build

```diff
@@ -343,33 +367,73 @@ checkEnv()
   #
   printHeader "Checking environment configuration"
 
-
-  if [ ! -z "$ZOPEN_FORCE_TYPE" ]; then
-    # user specified, so normalize in upper case
-    ZOPEN_TYPE=$(echo "${ZOPEN_FORCE_TYPE}" | tr '[a-z]' '[A-Z]')
-  fi
-
-  if [ "${ZOPEN_TYPE}x" = "TARBALLx" ]; then
-    if [ "${ZOPEN_TARBALL_URL}x" = "x" ]; then
-      export ZOPEN_TARBALL_URL="${ZOPEN_URL}"
-    fi
-    if [ "${ZOPEN_TARBALL_DEPS}x" = "x" ]; then
-      export ZOPEN_TARBALL_DEPS="${ZOPEN_DEPS}"
-    fi
-    printHeader "Using tarball environment"
-  elif [ "${ZOPEN_TYPE}x" = "GITx" ]; then
-    if [ "${ZOPEN_GIT_URL}x" = "x" ]; then
-      export ZOPEN_GIT_URL="${ZOPEN_URL}"
-    fi
-
-    if [ "${ZOPEN_GIT_DEPS}x" = "x" ]; then
-      export ZOPEN_GIT_DEPS="${ZOPEN_DEPS}"
+  if [ ! -z "$ZOPEN_BUILD_LEVEL" ]; then
+    if [ "${ZOPEN_BUILD_LEVEL}x" = "DEVx" ]; then
```

Comment:
i am assuming somewhere before this, the ZOPEN_BUILD_LEVEL has been uppercased to normalize it?

Reply:
Yes, it is normalized in setDefaultBuild()